src/main/java/com/microsoft/azure/relay/ClientWebSocket.java

```diff
@@ -339,8 +352,14 @@ public void onClose(Session session, CloseReason reason) {
 	}
 
 	@OnError
-	public void onError(Session session, Throwable cause) {
-		RelayLogger.throwingException(cause, this);
+	public void onError(Throwable cause) {
```

Comment:
>public void onError(Throwable cause) [](start = 1, length = 36)

I see the onError signature changed. Are both of these valid (or at least the latest one)?

Reply:
It should not make a difference, both are valid. I haven't observed any difference either.

---
In reply to: [354519842](https://github.com/Azure/azure-relay-java/pull/58#discussion_r354519842) [](ancestors = 354519842)

Reply:
As long as it gets called it's good.

---
In reply to: [354522105](https://github.com/Azure/azure-relay-java/pull/58#discussion_r354522105) [](ancestors = 354522105,354519842)